README.md

```diff
@@ -1,6 +1,12 @@
 # Image Actions
 
+<<<<<<< HEAD
+Image Actions automatically compress JPEG, PNG and WebP images in GitHub Pull Requests within the same repository.
+
+It can also run in "Compress Only" mode to allow this to be used for non-Pull Requests, and also for Pull Requests from forks.
+=======
 Image Actions automatically compresses JPEG, PNG and WebP images in GitHub Pull Requests.
+>>>>>>> upstream/master

```

Comment:
There's enough documentation lower in the readme, could you remove this from the introduction section?

Reply:
Done.